facetapi.module

```diff
@@ -1532,8 +1570,8 @@ function _ctools_export_unpack_object($schema, $data, $object = 'stdClass') {
 }
 
 function ctools_export_load_object($table, $type = 'all', $args = array()) {
```

Comment:
These `ctools_**()` functions should really be renamed or simplified and replaced.